bullet_train-themes/app/views/themes/base/attributes/_number.html.erb

```diff
@@ -1,10 +1,15 @@
 <% object ||= current_attributes_object %>
 <% strategy ||= current_attributes_strategy || :none %>
 <% url ||= nil %>
+<% format ||= nil %>
 <% options ||= {} %>
 
 <% if object.send(attribute).present? %>
   <%= render 'shared/attributes/attribute', object: object, attribute: attribute, strategy: strategy do %>
-    <%= object.send(attribute) %>
+    <% if format %>
+      <%= sprintf(format, object.send(attribute)) %>
+    <% else %>
+      <%= object.send(attribute) %>
```

Comment:
I recommend using `public_send` instead of `send` whenever possible. Most of the time it will be functionally equivalent, but it might help avoid some subtle bugs.

Reply:
Sure, since we use it in different places across the repository, I would prefer to change everything at once as opposed to just applying it to this PR alone.

I'm not too familiar with the bugs or subtle differences, would you mind elaborating a bit? Thank you!

Reply:
@gazayas I had this PR changing `send` to `public_send` anywhere I could find. But I have no idea why the test is failing there https://github.com/bullet-train-co/bullet_train-core/pull/307

Reply:
I'd be happy to help sort out the test, but I'm honestly not sure how to run that test locally.

Reply:
I remember having this same issue before and @andrewculver managed to fix it somehow :)

Reply:
@newstler No problem, I'm glad I could help! I also went ahead and used `public_send` in this PR :smiley: